selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java

```diff
@@ -175,6 +180,10 @@ public void checkWorkspaceSelectingByCheckbox() throws Exception {
     workspaces.selectWorkspaceByCheckbox(javaWorkspaceName);
     workspaces.waitWorkspaceCheckboxEnabled(javaWorkspaceName);
     workspaces.waitWorkspaceCheckboxEnabled(blankWorkspaceName);
+
+    // for avoid of failing in the multi-thread mode
```

Comment:
it could be useful for reader to know what kind of fail is `clickOnUnexpectedWorkspacesCheckboxes` is preventing